packages/amplify-e2e-core/src/init/headless-types.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Shape of awscloudformation object within `--profile` payload for init/pull
+ */
```

Comment:
do we have a similar type in prod code base?

Reply:
we don't have an exact corollary but we have some types [here](https://github.com/aws-amplify/amplify-cli/blob/dev/packages/amplify-provider-awscloudformation/src/configuration-manager.ts#L29-L50) that define the credential config (plus other values that it can have)